cmd/manager/main.go

```diff
@@ -193,6 +202,9 @@ func main() {
 		},
 	)
 
+	klusterletconfigInformer := klusterletconfiginformer.NewSharedInformerFactory(
```

Comment:
Inspired by this PR: https://github.com/open-cluster-management-io/ocm/pull/232 We choose to use `InformerFacotory` here.